src/main/scala/nl/knaw/dans/easy/fsrdb/CLI.scala

```diff
@@ -26,7 +26,7 @@ object CLI {
     val conf = new Conf(args, props)
     implicit val settings = Settings(
       fedoraCredentials = new FedoraCredentials(conf.fedora(), conf.user(), conf.password()),
-      postgresURL = conf.db(), conf.dataset())
+      postgresURL = conf.db(), conf.input.get, conf.datasets.get)
```

Comment:
It would be cleaner to follow the named parameter style for all the fields of the Settings object.